src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -988,7 +988,7 @@ public boolean equals(Object obj) {
 
             FileStatus other = (FileStatus) obj;
 
-            if (!other.equals(this)) {// compare the path
+            if (!this.getPath().equals(other.getPath())) { // compare the path
```

Comment:
The comment is now unnecessary -- it's now obvious what it's doing.

Reply:
removed the comment.